value-fixture/src/org/immutables/fixture/annotation/ExampleModelWithSelfApplyingAnnotation.java

```diff
@@ -0,0 +1,15 @@
+package org.immutables.fixture.annotation;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import nonimmutables.SelfApplyingAnnotation;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@Value.Style(passAnnotations = {SelfApplyingAnnotation.class})
```

Comment:
I had to add this for the test to pass (I somehow thought I had it passing before - not sure what happened). It doesn't really matter though, since what we want to test is the fact that it compiles fine, though I'm not sure how to best set this up